scripts/analytics/src/modules/cli/script.ts

```diff
@@ -0,0 +1,88 @@
+import { exec } from "node:child_process";
+import { promisify } from "node:util";
+
+import { config } from "../../libs/modules/config/config.js";
+import { logger } from "../../libs/modules/logger/logger.js";
+import { COMMIT_REGEX } from "./libs/constants/constants.js";
+import {
+	formatDate,
+	getFetch,
+	getShortLogCmd,
+} from "./libs/helpers/helpers.js";
+import {
+	type ActivityLogCreateItemRequestDto,
+	type CommitStatistics,
+	type ScriptParameters,
+} from "./libs/types/types.js";
+
+const execAsync = promisify(exec);
+
+async function collectAndSendStats({
```

Comment:
This script should be a separate module with methods, like src/modules/analytics (similar to what we have in the backend, but we only will have `.service`).